src/odin/maneuver.cc

```diff
@@ -31,84 +31,6 @@ const std::unordered_map<int, std::string>
                               {static_cast<int>(Maneuver::RelativeDirection::kKeepLeft),
                                "Maneuver::RelativeDirection::kKeepLeft"}};
 
-const std::string& DirectionsLeg_Maneuver_Type_Name(int v) {
```

Comment:
it is used only if `#ifdef LOGGING_LEVEL_TRACE`, so moved it under appropriate `#ifdef`